components/places/src/api/places_api.rs

```diff
@@ -16,7 +17,11 @@ use std::sync::{
     atomic::{AtomicBool, AtomicUsize, Ordering},
     Arc, Mutex, Weak,
 };
-use sync15::{telemetry, ClientInfo};
+use sync15::{telemetry, MemoryCachedState};
+
+// Not clear if this should be here, but this is the "global sync state"
+// which is persisted to disk and reused for all engines.
+pub const GLOBAL_STATE_META_KEY: &str = "global_sync_state_v2";
```

Comment:
Mind also adding a note that this is stored globally because it's only ever round-tripped, not changed or otherwise used by the engines?